lib/src/main/java/com/tbruyelle/rxpermissions/RxPermissions.java

```diff
@@ -249,6 +252,21 @@ public boolean isGranted(String permission) {
     }
 
     /**
+     * Returns true if the permission is already granted.
+     * <p>
+     * Always true if SDK &lt; 23.
+     */
+    @SuppressWarnings("WeakerAccess")
+    public static boolean isGranted(Context context, String permission) {
+        return !isMarshmallowImplementation() || isGrantedImplementation(context, permission);
```

Comment:
@Luolc we can also create a Observable<Boolean> version for these static methods, like `Observable.fromCallable(() -> isGranted.(context, permission));`